asian-recipes.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Asian Chicken Recipes</title>
+  </head>
+  <body>
+    <center>
+      <h1>Asian style chicken</h1>
+
+      <a
+        href="/Users/mz/Documents/bootcamp/m1/day1/recipe-site-bootcamp/kung-pao-chicken.html"
+        ><h2>Kung pao chicken</h2></a
+      >
+
+      <a
+        href="/Users/mz/Documents/bootcamp/m1/day1/recipe-site-bootcamp/chicken-pho.html"
+        ><h2>Chicken pho</h2></a
+      >
+
+      <a
+        href="/Users/mz/Documents/bootcamp/m1/day1/recipe-site-bootcamp/butter-chicken.html"
+        ><h2>Butter chicken</h2></a
+      >
+
+      <img
+        src="/Users/mz/Documents/bootcamp/m1/day1/recipe-site-bootcamp/asian-food.jpeg"
+        alt="asian-food"
```

Comment:
I would write the alt tag in a normal english way. "Asian food" - that way screen readers can pick it up properly or it will be displayed if the image is unavailable